src/components/pages/Users/Users.js

```diff
@@ -842,16 +842,14 @@ const Users = () => {
           <tbody {...getTableBodyProps()}>
             {page.map((row) => {
               prepareRow(row)
-
+              const { key: _, ...restRowProps } = row.getRowProps()
               return (
-                <Tr key={row.id} {...row.getRowProps()}>
+                <Tr key={row.id} {...restRowProps}>
                   {row.cells.map((cell) => {
+                    const { key: _, ...restCellProps } = cell.getCellProps()
```

Comment:
using `key: _` to ensure that the key prop is not included in the object of remaining props that are spread into the JSX element